package.json

```diff
@@ -58,5 +58,8 @@
     "suiteNameTemplate": "{filepath}",
     "classNameTemplate": "{classname}",
     "titleTemplate": "{title}"
+  },
+  "dependencies": {
+    "base64-js": "^1.5.1"
   }
```

Comment:
hmmm, can we avoid this?

Right now we have **no dependencies** which is awesome.

Can we not stick with native code?

Reply:
Removed the dep. We now check for Node and use the `Buffer` class to do the encoding/decoding. Use browser provided `btoa` and `atob` if not node.